src/Microsoft.DotNet.Tools.Publish/PublishCommand.cs

```diff
@@ -14,124 +14,74 @@
 
 namespace Microsoft.DotNet.Tools.Publish
 {
-    public class Program
+    public class PublishCommand
     {
-        public static int Main(string[] args)
-        {
-            DebugHelper.HandleDebugSwitch(ref args);
+        public string ProjectPath;
+        public string Configuration;
+        public string OutputPath;
+        public string Framework;
+        public string Runtime;
+        public NuGetFramework NugetFramework;
+        public IEnumerable<ProjectContext> ProjectRidFxPermutations;
```

Comment:
Tell me why it is not good practice and I will change it. Currently this feels like it is:
`public string ProjectPath;` vs `public string ProjectPath { get; set; }`. The first one is shorter, rest of the code looks the same. If I start making this class immutable then we will have: a field, a property, an immutable class, an immutable class builder and we can just go forever like that and create another FizzBuzz Enterprise Edition (https://github.com/EnterpriseQualityCoding/FizzBuzzEnterpriseEdition)